<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>292</width>
    <height>155</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>292</width>
    <height>155</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>292</width>
    <height>155</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Здравствуйте!</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>273</width>
      <height>123</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>273</width>
      <height>123</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>273</width>
      <height>123</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter_5">
       <property name="minimumSize">
        <size>
         <width>271</width>
         <height>121</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>271</width>
         <height>121</height>
        </size>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QSplitter" name="splitter_4">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>93</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>93</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <widget class="QSplitter" name="splitter_3">
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>271</width>
           <height>44</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>19</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Ваше ФИО:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="doctorData">
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>20</height>
           </size>
          </property>
         </widget>
        </widget>
        <widget class="QSplitter" name="splitter_2">
         <property name="minimumSize">
          <size>
           <width>271</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>271</width>
           <height>44</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>19</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>19</height>
           </size>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Пароль:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="userpassword">
          <property name="minimumSize">
           <size>
            <width>271</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>271</width>
            <height>20</height>
           </size>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QSplitter" name="splitter">
        <property name="minimumSize">
         <size>
          <width>271</width>
          <height>23</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>271</width>
          <height>23</height>
         </size>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <widget class="QPushButton" name="regBtn">
         <property name="minimumSize">
          <size>
           <width>160</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>160</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>Зарегистрироваться</string>
         </property>
        </widget>
        <widget class="QPushButton" name="pushButton">
         <property name="minimumSize">
          <size>
           <width>106</width>
           <height>23</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>106</width>
           <height>23</height>
          </size>
         </property>
         <property name="text">
          <string>OK</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
